metrics/perplexity/README.md

```diff
@@ -0,0 +1,61 @@
+# Metric Card for Perplexity
+
+## Metric Description
+Given a model and an input text sequence, perplexity measures how likely the model is to generate the input text sequence. 
+
+## Intended Uses
+Any language generation task.
+
+## How to Use
+
+```python
+from datasets import load_metric
+exact_match_metric = load_metric("perplexity")
+results = exact_match_metric.compute(predictions=predictions, references=references)
```

Comment:
This doesn't look perplexity :P

Reply:
Whoops!! You're right, sorry about that! Will fix right now